src/constants/value_commit_v.rs

```diff
@@ -0,0 +1,3735 @@
+use super::{OrchardFixedBase, ValueCommitV, VALUE_COMMITMENT_PERSONALIZATION};
+use halo2::arithmetic::{CurveAffine, FieldExt};
+
+pub const PERSONALIZATION: &str = VALUE_COMMITMENT_PERSONALIZATION;
+
```

Comment:
```suggestion
pub const PERSONALIZATION: &str = VALUE_COMMITMENT_PERSONALIZATION;

```
I would prefer to delete this and use VALUE_COMMITMENT_PERSONALIZATION directly; I think that's clearer.